docs/user-guide/cli-using-initializing-team-configuration.md

```diff
@@ -16,7 +16,7 @@ You can use methods that are described in this article to initialize team config
 
     The `zowe config init` command ensures that your credentials are stored securely on your computer by default.
 
-    After you respond, the `zowe.config.json` team configuration file is added to your local `.zowe` directory. This location is the primary location where your mainframe service connection details are defined, such as host and port. You use Use this configuration file for the following procedures.
+    After you respond, the `zowe.config.json` team configuration file is added to your local `.zowe` directory. This location is the primary location where your mainframe service connection details are defined, such as host and port. You use this configuration file for the following procedures.
 
 After you create the configuration files, you can use a text editor or IDE, such as *Visual Studio Code*, to populate global profiles with connection details for your mainframe services.
 ## Creating team profiles by defining a connection to z/OSMF
```

Comment:
I don't think we should duplicate content. Steps 1 & 2 below are the same as steps 1 & 2 above. Also, I would position `auto-init` as the recommended method to use if API Mediation Layer is in place.